language/Portuguese_Brazil/strings.po

```diff
@@ -392,23 +395,23 @@ msgstr "Cores"
 
 #: src/frontend/mame/ui/custui.cpp:157 src/frontend/mame/ui/dirmenu.cpp:38
 msgid "Language"
-msgstr "Linguagem"
+msgstr "Idioma"
 
 #: src/frontend/mame/ui/custui.cpp:161
 msgid "Show side panels"
 msgstr "Mostrar painéis laterais"
 
 #: src/frontend/mame/ui/custui.cpp:173
 msgid "Custom UI Settings"
-msgstr "Configurações Customizadas do UI"
+msgstr "Configurações Personalizadas da IU"
 
 #: src/frontend/mame/ui/custui.cpp:223
 msgid "default"
 msgstr "padrão"
 
 #: src/frontend/mame/ui/custui.cpp:331
 msgid "UI Font"
-msgstr "Fonte do UI"
+msgstr "Fonte da IU"
 
 #: src/frontend/mame/ui/custui.cpp:336
 msgid "Bold"
```

Comment:
Brazilian computer users are used to the term "Bold" refering to darker than "Regular" font weigths. But if we want to use a proper portuguese word we could translate this to "Negrito". Keeping it as "Bold" would do no harm, though.